package.json

```diff
@@ -19,24 +20,26 @@
   },
   "homepage": "https://github.com/chingu-voyage7/Bears-Team-30#readme",
   "dependencies": {
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
-    "react-router-dom": "^4.3.1",
-    "webpack": "^4.26.1",
-    "webpack-cli": "^3.1.2",
+    "apollo-server": "^2.2.2",
+    "apollo-server-cache-redis": "^0.2.1",
```

Comment:
What is apollo-server-cache-redis for? Does it mean apollo needs redis to implement caching?

Reply:
yah -- I haven't actually set it up, but it'd be a local memory cache for queries. We could use other options, or not include one at all